src/05-derivatives/05-diffusion-derivatives.md

```diff
@@ -18,450 +18,700 @@ should be included in the pipeline documentation.
-| ModelURL         | OPTIONAL. URL to the implementation of the specific model utilized.                                                                  |
-| Parameters       | OPTIONAL. A dictionary of model parameters (see below).                                                                              |
-
-Parameters that may be stored within the JSON sidecar "Parameters" field depend on the particular model used:
-
--   `DTI` :
-
-    -   `FitMethod` : {`WLS`,`IWLS`,`OLS`,`NLLS`}
-    -   `OutlierRejection`: boolean
-    -   `RESTORESigma` : value
-
--   `DKI` and `WMTI`:
-
-    -   `FitMethod` : {`WLS`,`IWLS`,`OLS`,`NLLS`}
-
--   `CSD`:
-
-    -   `Tissue` : string
-    -   `SphericalHarmonicDegree` : value
-    -   `ResponseFunctionZSH` : values (1 row per unique *b*-value as listed in "`Shells`"; 1 column per even harmonic degree starting from zero)
-    -   `ResponseFunctionTensor` : values (vector of 4 values: three tensor eigenvalues, then reference b=0 intensity)
-    -   `SphericalHarmonicBasis` : {`MRtrix3`,`DESCOTEAUX`}
-    -   `NonNegativityConstraint` : {`soft`,`hard`}
-
--   `NODDI`:
-
-    -   `DPar` : value
-    -   `DIso` : value
-    -   `Lambda1` : value
-    -   `Lambda2` : value
-
--   `DSI` :
-
-    -   `GridSize` : value
-    -   `RStart` : value
-    -   `RStep` : value
-    -   `REnd` : value
-    -   `FilterWidth` : value
-
--   `CSA` :
-
-    -   `SphericalHarmonicOrder` : value
-    -   `Smoothing` : value
-    -   `Basis` : value
-
--   `SHORE` :
-
-    -   `RadialOrder` : value
-    -   `Zeta` : value
-    -   `LambdaN` : value
-    -   `LambdaL` : value
-    -   `Tau` : value
-    -   `ConstrainE0` : value
-    -   `PositiveConstraint` : value
-    -   `PosGrid` : value
-    -   `PosRadius` : value
-
--   `MAPMRI` :
-
-    -   `RadialOrder` : value
-    -   `LaplacianRegularization` : bool
-    -   `LaplacianWeighting` : value
-    -   `PositivityConstraint` : bool
-    -   `Tau` : value
-    -   `ConstrainE0` : value
-    -   `PositiveConstraint` : value
-    -   `PosGrid` : value
-    -   `PosRadius` : value
-    -   `AnisotropicScaling` : bool
-    -   `EigenvalueThreshold` : value
-    -   `PosGrid` : value
-    -   `BvalThreshold` : value
-    -   `DTIScaleEstimation` : bool
-    -   `StaticDiffusivity` : value
-
--   `FORECAST` :
-
-    -   `Sphere` : value
-    -   `DecAlg` : value
-    -   `LambdaLb` : value
-    -   `SphericalHarmonicsOrder` : value
-
--   `fwDTI` :
-
-    -   `FitMethod` : {"WLS","NLLS"}
+-  File "`<source_keywords>[_space-<space>][_desc-<label>]_<model>.json`"
+   provides basic model information and [input](#paramdef-input)
+   model parameters.
 
--   `BEDPOSTX` :
+-  Images "`<source_keywords>[_space-<space>][_desc-<label>]_parameter-<intparam*>_<model>.nii[.gz]`"
+   MAY be defined, in order to provide additional [extrinsic](#paramdef-extrinsic)

```

Comment:
Should `extrinsic` be `intrinsic` here? Or, if not the case, then the filename should refer to `extparam`. Either way I think we are missing either the `extparam` or `intparam` example.

Reply:
Corrected in 90fa5fa.